AzureTable/Microsoft.DataTransfer.AzureTable.FunctionalTests/Microsoft.DataTransfer.AzureTable.FunctionalTests.csproj

```diff
@@ -66,6 +66,18 @@
       <HintPath>..\..\packages\Microsoft.Data.Services.Client.5.8.3\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.OData.Core, Version=7.5.1.20914, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Microsoft.OData.Core.7.5.1\lib\portable-net45+win8+wpa81\Microsoft.OData.Core.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.OData.Edm, Version=7.5.1.20914, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Is this supposed to replace Microsoft.Data.Edm on line 57 or they both are needed? Same for Microsoft.Data.OData - if I understand correctly, this is a new version of that library/package. Do we still need both? There might be some conflicts between dependencies (if they have any shared dependencies with different versions, like JSON.NET, for example).

Reply:
you're correct, they're both present in the build output previously, so I am adding them to not break any use-cases. 
They're both dependencies to Microsoft.Azure.CosmosDB.Table.2.0.0 (newer version)